src/Kerbero.Common/Errors/CommonErrors/InvalidParametersError.cs

```diff
@@ -6,7 +6,7 @@ public class InvalidParametersError: KerberoError
 {
 	public InvalidParametersError(object parameter,
 		string? message = "There are missing or wrong parameter in the request: ") : base(
-		message + parameter.ToString())
+		message + parameter + ".")
```

Comment:
Why? Do we expect everyone that builds an `InvalidParametersError` to know that a `.` will be appended to their message?

Reply:
and how i can hide it?